app/serializers/serialize_mentor_discussion.rb

```diff
@@ -0,0 +1,55 @@
+class SerializeMentorDiscussion
+  include Mandate
+
+  initialize_with :discussion, :context
+
+  def call
+    return if discussion.blank?
+
+    {
+      id: discussion.uuid,
+      mentor: {
+        avatar_url: discussion.mentor.avatar_url,
+        handle: discussion.mentor.handle
+      },
+      student: {
+        avatar_url: discussion.student.avatar_url,
+        handle: discussion.student.handle
+      },
+      track: {
+        title: discussion.track.title,
+        icon_url: discussion.track.icon_url
+      },
+      exercise: {
+        title: discussion.exercise.title,
+        icon_url: discussion.exercise.icon_url
+      },
+      is_finished: context == :student ? discussion.finished_for_student? : discussion.finished_for_mentor?,
```

Comment:
Could you pull all of these out into methods on the serializer pls.

e.g. 
```ruby
def is_finished
  context == :student ? discussion.finished_for_student? : discussion.finished_for_mentor?,
end
```

This one is ok but the other if/else/end ones make reading this all quite hard :)

Reply:
Thanks. I've done this :)